tutorials/polkadot-sdk/parachains/build-custom-pallet/pallet-unit-testing.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Pallet Unit Testing
+description: Discover how to create thorough unit tests for pallets built with the Polkadot SDK, using a custom pallet as a practical example.
+---
+
+# Pallet Unit Testing
+
+## Introduction
+
+You have learned how to create a custom pallet in the [Build the Pallet](/tutorials/polkadot-sdk/parachains/build-custom-pallet/){target=\_blank} tutorial; now you will see how to test the pallet to ensure that it works as expected. As stated in the [Pallet Testing](/develop/parachains/customize-parachain/pallet-testing/){target=\_blank} article, unit testing is crucial for ensuring the reliability and correctness of pallets in Polkadot SDK-based blockchains. Comprehensive testing helps validate pallet functionality, prevent potential bugs, and maintain the integrity of your blockchain logic.
+
+This tutorial will guide you through creating a unit testing suite for a custom pallet built in the previous tutorial, covering essential testing aspects and steps.
```

Comment:
```suggestion
This tutorial will guide you through creating a unit testing suite for a custom pallet built in the [previous tutorial](/tutorials/polkadot-sdk/parachains/build-custom-pallet/){target=\_blank}, covering essential testing aspects and steps.
```

I know it's linked to in the previous paragraph, but doesn't hurt to add it here too